extensions/eclipse-che-theia-remote-api/src/browser/oauth-utils.ts

```diff
@@ -58,7 +58,7 @@ export class OauthUtils {
     });
   }
 
-  async getUserToken(): Promise<string | undefined> {
+  private async getUserToken(): Promise<string | undefined> {
```

Comment:
it seems you can't:
src/browser/che-quick-open-workspace.ts(91,41): error TS2341: Property 'getUserToken' is private and only accessible within class 'OauthUtils'.

Reply:
Thanks, it is strange, but I've tried to build it locally and it was ok. Anyway, removed the `private` imodifier.